fairing/backends/backends.py

```diff
@@ -34,37 +35,40 @@ class KubernetesBackend(BackendInterface):
 
     def __init__(self, namespace=None):
         self._namespace = namespace
-    
+
     def get_builder(self, preprocessor, base_image, registry, needs_deps_installation=True, pod_spec_mutators=None):
         if fairing.utils.is_running_in_k8s():
+            logging.info("Using ClusterBuilder")
             return ClusterBuilder(preprocessor=preprocessor,
                                   base_image=base_image,
                                   registry=registry,
                                   pod_spec_mutators=pod_spec_mutators,
                                   namespace=self._namespace)
         elif ml_tasks_utils.is_docker_daemon_exists():
+            logging.info("Using Docker builder")
             return DockerBuilder(preprocessor=preprocessor,
                                  base_image=base_image,
                                  registry=registry)
         elif not needs_deps_installation:
+            logging.info("Using AppendBuilder")
```

Comment:
Let's put the log messages in the builders themselves so it doesn't have to be replicated everywhere we instantiate them.

Reply:
Done.